.gitignore

```diff
@@ -18,6 +18,7 @@ group_vars/*.yml
 ceph-ansible.spec
 *.retry
 *.pytest_cache
+.vscode/
```

Comment:
Just do global exclude locally since these files are not related to the ceph-ansible repo but your local coding env`git config --global core.excludesfile ~/.gitignore_global`.